restapi/stories.go

```diff
@@ -57,12 +73,74 @@ func (api api) CreateStories(w http.ResponseWriter, r *http.Request) render.Rend
 
 	for i := 0; i < len(stories); i++ {
 		city := stories[i].CurrentCity
-		coordinates, err := api.locationFinder.GetCityCenter(city)
+		stories[i].CurrentCity = strings.Title(strings.ToLower(city))
+
+		// get number of stories with current city in db
+		var prevStories []models.Story
+		err := api.database.Where("current_city=?", city).Find(&prevStories).Error
+		if err != nil {
+			api.logger.Infof("A")
+			// default to randomizing if there is an error
+			stories[i].Latitude, stories[i].Longitude, err = api.randomizePinLocation(city)
```

Comment:
You seem to default to randomizing in a few places so its best to extract this into a helper function

Reply:
moved